src/material-experimental/mdc-checkbox/checkbox.ts

```diff
@@ -259,6 +288,11 @@ export class MatCheckbox implements AfterViewInit, OnDestroy, ControlValueAccess
     // @breaking-change 10.0.0: Remove this after the `_clickAction` parameter is removed as an
     // injection parameter.
     this._clickAction = this._clickAction || this._options.clickAction;
+    console.log('potato!!!!!!!!!!!!!!!');
```

Comment:
remove this

Reply:
Whoops, I'm not sure how it even is there. Sorry!